<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget {
background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
border-image:rgb(44, 37, 255);
border : rgb(44, 37, 255);
background-repeat: rgb(44, 37, 255);
}</string>
  </property>
  <widget class="QFrame" name="Menu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Menu{
background-color:rgb(218, 84, 84);
border-image:rgb(44, 37, 255);
border : rgb(0, 0, 0);
background-repeat: rgb(44, 37, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioserver">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#radioserver{
background-color:rgb(125, 142, 255);
border-image:rgb(44, 37, 255);
border : rgb(0, 0, 0);
background-repeat: rgb(44, 37, 255);
}</string>
    </property>
    <property name="text">
     <string>Ser&amp;ver</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioclient">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#radioclient{
background-color:rgb(125, 142, 255);
border-image:rgb(44, 37, 255);
border : rgb(0, 0, 0);
background-repeat: rgb(44, 37, 255);
}</string>
    </property>
    <property name="text">
     <string>C&amp;lient</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>311</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="connectionMode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#1f26ad;&quot;&gt;        Start Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Client">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>16</width>
     <height>321</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#Client{
background-color:rgb(255, 99, 99);
border-image:rgb(44, 37, 255);
border : rgb(0, 0, 0);
background-repeat: rgb(44, 37, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ClientId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Adresse:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClientIPText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClientPort">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClientPortText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Abord</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CLientConnect">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClientName">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NicktName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Nickname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Color">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ClientColor">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cyan</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Server">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>16</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Server{
background-color:rgb(255, 99, 99);
border-image:rgb(44, 37, 255);
border : rgb(0, 0, 0);
background-repeat: rgb(44, 37, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ServerId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Adresse:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ServerIPText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="PortServer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ServerPortText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="GameParameters">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Parameters:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ServerAbord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Abord</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ServerConnect">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="ClientId_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Nickname:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ServerName">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClientId_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ServerColor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cyan</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="ServerFirst">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ser&amp;ver First</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RandomFirst">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rando&amp;m</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ClientFirst">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>C&amp;lient First</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="width">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>52</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="height">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>52</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="Spielgrosse_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Spielgrosse_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Spielgrosse_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Round:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Round">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>52</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
